package.json

```diff
@@ -101,6 +101,7 @@
     "babel-jest": "^26.2.2",
     "babel-loader": "^8.1.0",
     "css-loader": "^4.2.0",
+    "cssom": "^0.4.4",
```

Comment:
Required so we can load a style sheet object from CSS generated from a SASS file. The style sheet is used to auto-generate Storybook stories.